mlir/utils/jenkins/Jenkinsfile

```diff
@@ -494,6 +560,9 @@ pipeline {
                 }
                 stage("Build MLIR") {
                     steps {
+                        // Clean up build to disable static library and allow
+                        // mlir-rocm-runner
+                        sh 'rm -rf build'
```

Comment:
This will also delete the performance results created in the stage "Performance Test: Tuned vs Untuned". They are needed to create performance html.